spec/factories/clearance.rb

```diff
@@ -4,7 +4,18 @@
   end
 
   factory :user do
-    email
+    email { generate(:email) }
```

Comment:
This change is not necessary. The email sequence works without it
<img width="514" alt="image" src="https://github.com/user-attachments/assets/7d861d93-c358-46c5-b745-9b642b2d4d91" />